cart/src/App.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from "react";
+import Cart from "./components/Cart";
+import Home from "./components/Home";
+import NavBar from "./components/Navbar";
+
+const App = () => {
+  const [showScreen, setShowScreen] = useState(true);
+  const [cartArray, setCartArray] = useState([]);
+  const [totalPrice, setTotalPrice] = useState(0);
+
+  const addItemToCart = (items) => {
```

Comment:
I think the variable name  `items` here is confusing, implying that an array is passed. Pls rename